config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,136 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
These will be addressed in a follow-up ticket: https://cm-jira.usa.gov/browse/SRCH-3413